### PR TITLE
Fix/mc 25  starting page redo

### PR DESCRIPTION
**Pull request by MatejaC**

### CODE REVIEW

File: src/components/SearchBox.tsx [hunk 1]
```diff
@@ -66,27 +66,16 @@ const SearchBox = ({ setSearchResults }: SearchBoxProps) => {
...
-        [theme.breakpoints.up('lg')]: {
-          maxWidth: '80%',
-        },
-        [theme.breakpoints.up('xl')]: {
-          maxWidth: '60%',
-        },
-      })}
     >
-      <Box flexGrow={3}>
+      <Box minWidth='40vw' width='105%'>
```
[the remark this answers is not in the document]
Comment (e1401) [reply src/components/SearchBox.tsx:78]: Check Slack and the video example there because it is over 10mb limit

**Review by e1401 — CHANGES_REQUESTED** [review]

**Comment by MatejaC** [comment]
Tnx! :)

**Review by e1401 — APPROVED** [review]
✅

**Comment by e1401** [comment]
Closed due to inactivity.

**Pull request closed without merging**